app/views/list/_list_row_template.slim

```diff
@@ -0,0 +1,10 @@
+tr.request
+  td.request-checkbox
+    input[type="checkbox"]
```

Comment:
Input should have a name to allow for bulk actions to be submitted via a form. I'd suggest `name="requests[][selected]"` with a separate hidden field with the name `requests[][id]` to track which records were selected.